src/Components/Publishing/Sections/Text.tsx

```diff
@@ -85,14 +86,39 @@ export class Text extends Component<Props, State> {
 
     if (node.name === "a" && this.shouldShowTooltipForURL(node)) {
       const href = node.attribs.href
-      const text = node.children[0] && node.children[0].data
-
-      if (text) {
-        return (
-          <LinkWithTooltip key={href + index} url={href} color={color}>
-            {text}
-          </LinkWithTooltip>
-        )
+      const linkNode = get(node, n => n.children[0].data && n.children[0], {})
+
+      if (linkNode.data) {
+        const props = { key: href + index, url: href, color }
+        const next = linkNode.parent && linkNode.parent.next
+        const text = linkNode.data
+        const apostropheRe = /[’'][a-zA-Z]/
```

Comment:
I believe this also happens with other kinds of punctuation, like commas or periods -- worth double checking these cases and potentially applying to punctuation generally.

Reply:
Since I'm not familiar with the general patterns inside of editorial I'd rather wait until we have an explicit need to update (via example), and then we can update here as needed.

Reply:
I think you could pretty easily test this by inserting `.` or `,` or `;` rather than adding `'s` in your fixture.

Reply:
Updated JIRA ticket description with potential follow-up work (outside of this PR).